util/flag.go

```diff
@@ -0,0 +1,51 @@
+// Package util implements handy primitives that are too small to be placed
```

Comment:
nice! let's move it to https://github.com/flashbots/go-utils/blob/main/cli/cli.go ?